### PR TITLE
Beandev

### PR DESCRIPTION
**Pull request by lovendatj**
Create page component for future implementation of the project page.

Added cardType tag to each project (regular, large). Currently no SCCS styling changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,32 @@
-# opensourcery
- 
+#  Open Sourcery
+> A collection of templates for creating applications and websites.
```
Comment (DonIsaac) [inline README.md]: wot

File: src/components/lib/ProjectCard.tsx [hunk 2]
```diff
@@ -17,12 +17,13 @@ export const ProjectCard = ({
...
     description,
     url,
+    cardType,
     stars,
     watches,
     forks
 }: ProjectCardProps) => (
         <div id={id} className="col-12 col-sm-6 col-xl-3 pb-4 project-card">
-            <div className="card" style={{ height: '100%' }}>
+            <div className={"card "+ cardType} style={{ height: '100%' }}>
```
Comment (DonIsaac) [inline src/components/lib/ProjectCard.tsx]: use the `classnames` package

File: src/hooks/useGithubRepos.ts [hunk 3]
```diff
@@ -4,6 +4,15 @@ import { throws } from "./util";
 // curl "https://api.github.com/search/repositories?q=stars:>0+user:opensourceryclub&sort=stars&order=desc"
 export type SortBy = "updated" | "forks" | "help-wanted-issues" | "stars";
 
+enum CardType {
+    Regular = "regular-card",
+    Large   = "large-card"
+};
+        
+const cardType = {
+    regular     :{type:CardType.Regular, maxChar:75},
```
Comment (DonIsaac) [inline src/hooks/useGithubRepos.ts]: make the card type the keys, e.g `[CardType.Regular]: 75`

File: src/hooks/useGithubRepos.ts [hunk 4]
```diff
@@ -31,6 +40,7 @@ export interface Project {
     stars: number;
     watches: number;
     forks: number;
+    cardType: string;
```
Comment (DonIsaac) [inline src/hooks/useGithubRepos.ts]: Why is this a prop? isn't it what's returned?

**Review by DonIsaac — CHANGES_REQUESTED** [review]

File: src/components/lib/ProjectCard.tsx (hunk 2 above)
Comment (lovendatj) [reply src/components/lib/ProjectCard.tsx]: How would I use the ```classnames``` for this?

I don't think it should be a class either. Would it be better to add it as an ID in the ```ProjectCards``` section?

File: src/hooks/useGithubRepos.ts [hunk 5]
```diff
@@ -31,6 +40,7 @@ export interface Project {
     stars: number;
     watches: number;
     forks: number;
+    cardType:CardType;
```
Comment (DonIsaac) [inline src/hooks/useGithubRepos.ts:48]: This should not be here as this parameter is never used

**Pull request closed without merging**